scipy/stats/_continuous_distns.py

```diff
@@ -2556,7 +2556,17 @@ class gamma_gen(rv_continuous):
 
     When :math:`a` is an integer, `gamma` reduces to the Erlang
     distribution, and when :math:`a=1` to the exponential distribution.
+    
+    Gamma distributions are sometimes parameterized with two variables,
+    with a probability density function of:
 
+    .. math::
+
+        f(x, \alpha, \beta) = \frac{\beta^\alpha}{\Gamma(\alpha)} x^{\alpha - 1} e^{-\beta x }
```

Comment:
This renders nicely in the doc check
![Screenshot from 2020-08-05 23-56-55](https://user-images.githubusercontent.com/10172976/89500980-70b1c500-d777-11ea-95ca-898791718cca.png)

---
I think it could look nicer if both formulas match visually.
Using `exp` or `e^` in both and having all terms (Gamma * x^... * e^...) above the fraction could be a nice improvement.